### PR TITLE
Implementing CM5 into cclib

### PR DESCRIPTION
**Pull request by GomaruSai**
This PR aims at implementing CM5 for cclib, along with an example ORCA output file. It has several limitations though, which are addressed in the opening comment of cm5.py

This resolves issue #676

### CODE REVIEW

**Comment by GomaruSai** [comment]
The Travis Build failed because there I have committed multiple interdependant files simultaneously. However, I have tested this on my own machine with all the changed files in my cclib package and can confirm that it does work as intended.

**Comment by langner** [comment]
> The Travis Build failed because there I have committed multiple interdependant files simultaneously. However, I have tested this on my own machine with all the changed files in my cclib package and can confirm that it does work as intended.

That does not appear to be the case. Travis [says](https://travis-ci.com/github/cclib/cclib/jobs/326307882#L867) the some unit test started failing:

```
basicORCA4.1/dvb_sp.out: Customized restricted single point unittest
testatomcharges (test.data.testSP.OrcaSPTest)Traceback (most recent call last):
  File "/home/travis/build/cclib/cclib/test/data/skip.py", line 18, in testwrapper
    testfunc(self, *args, **kwargs)
  File "/home/travis/build/cclib/cclib/test/data/skip.py", line 30, in testwrapper
    testfunc(self, *args, **kwargs)
  File "/home/travis/build/cclib/cclib/test/data/skip.py", line 30, in testwrapper
    testfunc(self, *args, **kwargs)
  File "/home/travis/build/cclib/cclib/test/data/skip.py", line 18, in testwrapper
    testfunc(self, *args, **kwargs)
  File "/home/travis/build/cclib/cclib/test/data/testSP.py", line 67, in testatomcharges
    self.assertEqual(len(charges), self.data.natom)
AssertionError: 18 != 20
```

File: data/ORCA/basicORCA4.2/water_hpa.inp [hunk 1]
```diff
@@ -0,0 +1,9 @@
+%output
+Print[P_Hirshfeld] 1
```
Comment (langner) [inline data/ORCA/basicORCA4.2/water_hpa.inp:2]: Can we just put this into the default single-point logfile?

**Comment by langner** [comment]
Given there has been no follow up activity, @berquist do you think this is worth keeping around and trying to clean up for v1.6.5?

**Comment by berquist** [comment]
Yes, I definitely want this.

**Comment by berquist** [comment]
Since the contributing repository is not stable and we'd like to clean things up, this is replaced by #1136.

**Pull request closed without merging**